frontend/indigenousapprenticeship/src/js/components/Dialog/index.ts

```diff
@@ -0,0 +1,3 @@
+export { default as ApplyDialog } from "./ApplyDialog";
```

Comment:
This is super-nitpicky so feel free to tell me to buzz off or open a new issue. :grin:  When opening a dialog the page behind it jumps down and right a few pixels which is a little disorienting, especially on a big monitor.  Do you have any ideas if it is possible to avoid that?

Reply:
Watch the placement of the girl's eyes and the right edge of the image compared to the storybook controls here
![Peek 2022-05-03 09-04](https://user-images.githubusercontent.com/8978655/166458301-696a9aa4-9ddb-47f3-986b-ba9fe3b45f65.gif)
:

Reply:
Okay, I see it is clearly happening in that gif but I am having trouble reproducing the issue on my end. Were you able to get this to happen outside storybook?

Reply:
Oh you're right - it only happens in Storybook.  OK, I'm not worried about that.